itests/hive-unit/src/test/java/org/apache/hive/jdbc/TestJdbcWithMiniLlapVectorArrowBatch.java

```diff
@@ -320,7 +320,7 @@
 
   // test with legacy avro files
   // similar to ql/src/test/queries/clientpositive/avro_legacy_mixed_timestamp.q
-  @Test public void testAvroLegacyMixedTimestamps() throws Exception {
+  @Test(timeout = 12000000) public void testAvroLegacyMixedTimestamps() throws Exception {
```

Comment:
Is this because you got test failures for this?

Reply:
Reverted the change